src/Widgets/ColumnContainerWidget/ColumnsEditorTab.tsx

```diff
@@ -193,11 +193,11 @@ const ColumnsEditor = connect(
           labels={['No', 'Yes']}
           onChange={() =>
             widget.update({
-              disableResponsiveAdaption: !disableResponsiveAdaption,
+              disableResponsiveAdaption: !!responsiveAdaption,
```

Comment:
* [ ] FYI `!!` on a now boolean is not needed. Maybe we can  make improve the variable names for more clarity, or keep the original `disableResponsiveAdaption` around.